usr.sbin/sysinst/msg.mi.pt

```diff
@@ -0,0 +1,1528 @@
+ *  $3 = optional install flag          (I)nstall,
+ *  $4 = additional flags description	(B)ootable
+ *  $5 = total size			2TB
+ *  $6 = free size			244MB
+ */
+message fspart
+{Agora temos suas parti�oes de $2 por $0 abaixo.
+Esta � sua �ltima chance de alter�-las.
+
+Flags: $3(N)ewfs$4. Tamanho total: $5, livre: $6}
+
+message ptnheaders_start	{Inicio}
+message ptnheaders_end		{Fim}
+message ptnheaders_fstype	{FS}
+
+message partition_sizes_ok
+{Tamanhos de parti�ao ok}
+
+message edfspart
+{Os valores atuais para esta parti�ao sao
+exibidos abaixo.
+
+Selecione o campo que deseja alterar:}
+
+message ptn_newfs		{newfs}
+message ptn_mount		{mount}
+message ptn_mount_options	{mount options}
+message ptn_mountpt		{mount point}
+
+message toggle
+{Alternar}
+
+message restore
+{Restaurar valores originais}
+
+message Select_the_type
+{Selecione o tipo}
+
+message other_types
+{outros tipos}
+
+/* Called with:				Example
+ *  $0 = valid partition shortcuts	a-e
+ *  $1 = maximum allowed		4292098047
+ *  $2 = size unit			MB
+ */
+message label_size_head
+{Valores especiais que podem ser inseridos para o valor do tamanho:
+    -1:   usar at� o final}
+
+/* Called with:				Example
+ *  $0 = valid partition shortcuts	a-e
+ *  $1 = maximum allowed		4292098047
+ *  $2 = size unit			MB
+ */
+message label_size_part_hint
+{   $0:   usar at� a parti�ao fornecida}
+
+
+/* Called with:				Example
+ *  $0 = valid partition shortcuts	a-e
+ *  $1 = maximum allowed		4292098047
+ *  $2 = size unit			MB
+ */
+message label_size_tail			{Tamanho (max $1 $2)}
+
+/* Called with:				Example
+ *  $0 = valid partition shortcuts	a-e
+ *  $1 = valid free space shortcuts	f-h
+ *  $2 = size unit			MB
+ */
+message label_offset_head
+{Valores especiais que podem ser inseridos para o valor de deslocamento:
+    -1:   come�a no in�cio}
+
+/* Called with:				Example
+ *  $0 = valid partition shortcuts	a-e
+ *  $1 = valid free space shortcuts	f-h
+ *  $2 = size unit			MB
+ */
+message label_offset_part_hint
+{   $0:   come�a no final da parti�ao fornecida}
+
+/* Called with:				Example
+ *  $0 = valid partition shortcuts	a-e
+ *  $1 = valid free space shortcuts	f-h
+ *  $2 = size unit			MB
+ */
+message label_offset_space_hint
+{   $1:   comece no in�cio do espa�o livre fornecido}
+
+/* Called with:				Example
+ *  $0 = valid partition shortcuts	a-e
+ *  $1 = valid free space shortcuts	f-h
+ *  $2 = size unit			MB
+ */
+message label_offset_tail		{In�cio ($2)}
+
+message invalid_sector_number
+{N�mero mal formado}
+
+message packname
+{Por favor, insira um nome para o seu disco NetBSD}
+
+message lastchance
+{Ok, agora estamos prontos para instalar o NetBSD no seu disco r�gido (%s). Nada foi
+escrito ainda. Esta � sua �ltima chance de sair deste processo antes que algo
+seja alterado.
+
+Vamos continuar?
+}
+
+message disksetupdone
+{Ok, a primeira parte do procedimento est� conclu�da. O Sysinst
+escreveu um disklabel no disco de destino e fez newfs'ed e fsck'ed
```

Comment:
Perhaps "completou a criação e checagem das partições com newfs e fsck" would make more sense for people not that familiar with English?

Reply:
Isso ou `[...] destino e executou newfs e fsck`